### PR TITLE
Add Arcticblast

### PR DESCRIPTION
**Pull request by Spandamn**

### CODE REVIEW

File: data/mods/ssb/moves.ts [hunk 1]
```diff
@@ -451,6 +451,55 @@ export const Moves: {[k: string]: ModdedMoveData} = {
 		type: "Steel",
 	},
 
+	// Arcticblast
+	radiantburst: {
+		accuracy: true,
+		basePower: 10,
+		category: "Special",
+		desc: "Move causes Tapu Fini to become Briilliant if not, and vice versa. Move Mode depends on whether Tapu Fini is Briilliant or not.",
```
Comment (AnnikaCodes) [inline data/mods/ssb/moves.ts]:
```suggestion
		desc: "Move causes Tapu Fini to become Brilliant if not, and vice versa. Move Mode depends on whether Tapu Fini is Brilliant or not.",
```

File: data/mods/ssb/conditions.ts [hunk 2]
```diff
@@ -1194,6 +1206,42 @@ export const Conditions: {[k: string]: ModdedConditionData} = {
 			}
 		},
 	},
+	// Brilliant Condition for Arcticblast
+	brilliant: {
+		name: 'Brilliant',
+		duration: 5,
+		onStart(pokemon) {
+			pokemon.addVolatile("ingrain");
+		},
```
Comment (ghost) [inline data/mods/ssb/conditions.ts]: Just add the contents of the condition onto the mon

File: data/mods/ssb/conditions.ts [hunk 3]
```diff
@@ -1194,6 +1206,42 @@ export const Conditions: {[k: string]: ModdedConditionData} = {
...
+			return this.chainModify(1.5);
+		},
+		onModifySpe(spe, pokemon) {
+			return this.chainModify(1.5);
+		},
+		onUpdate(pokemon) {
+			if (pokemon.status === 'brilliant') {
+				this.add('-activate', pokemon, 'ability: Brilliant');
+				pokemon.removeVolatile('perishsong');
+			}
```
Comment (ghost) [inline data/mods/ssb/conditions.ts]: this isn't a nonvolatile status?

File: data/mods/ssb/moves.ts [hunk 4]
```diff
@@ -451,6 +451,55 @@ export const Moves: {[k: string]: ModdedMoveData} = {
...
+				this.add('-anim', source, 'Diamond Storm', target);
+			}
+		},
+		onModifyMove(move, pokemon) {
+			if (pokemon.volatiles['brilliant']) {
+				move.basePower = 180;
+				move.accuracy = 100;
+				move.type = "Fairy";
+				delete move.infiltrates;
+			}
```
Comment (ghost) [inline data/mods/ssb/moves.ts]: Can we do this the same way we do a move like future sight?

File: data/mods/ssb/conditions.ts (hunk 2 above)
Comment (Spandamn) [reply data/mods/ssb/conditions.ts]: I think adding ingrain will give the "Dug in roots" message

File: data/mods/ssb/conditions.ts (hunk 3 above)
Comment (Spandamn) [reply data/mods/ssb/conditions.ts]: https://github.com/HoeenCoder/Pokemon-Showdown/blob/master/data/moves.ts#L13326

File: data/mods/ssb/moves.ts (hunk 4 above)
Comment (Spandamn) [reply data/mods/ssb/moves.ts]: i felt like this was the best way to do it

Comment (ghost) [reply data/mods/ssb/moves.ts]: yeah I think we should 100% go the futuremove check route

Comment (ghost) [reply data/mods/ssb/moves.ts]: we should also have the ingrain effect implanted instead of an additional volatile

File: data/mods/ssb/moves.ts [hunk 5]
```diff
@@ -469,6 +469,55 @@ export const Moves: {[k: string]: ModdedMoveData & {gen?: number}} = {
...
+		accuracy: true,
+		basePower: 10,
+		category: "Special",
+		desc: "Move causes Tapu Fini to become Brilliant if not, and vice versa. Move Mode depends on whether Tapu Fini is Brilliant or not.",
+		shortDesc: "Move's Mode depends on whether Tapu Fini has the Brilliant condition or not.",
+		name: "Radiant Burst",
+		pp: 16,
+		priority: 0,
+		flags: {protect: 1},
+		onTryMovePriority: 100,
```
Comment (ghost) [inline data/mods/ssb/moves.ts]:
```suggestion
```

File: data/mods/ssb/conditions.ts [hunk 6]
```diff
@@ -1441,6 +1453,50 @@ export const Conditions: {[k: string]: ModdedConditionData} = {
...
+			if (status.id === 'perishsong') return null;
+			if (this.dex.getEffect(status.id).onDisableMove) return null;
+		},
+		onResidualOrder: 7,
+		onResidual(pokemon) {
+			this.heal(pokemon.baseMaxhp / 16);
+		},
+		onTrapPokemon(pokemon) {
+			pokemon.tryTrap();
+		},
```
Comment (ghost) [inline data/mods/ssb/conditions.ts]:
```suggestion
		},
```

**Pull request merged**